theme/templates/accounts/profile.html

```diff
@@ -162,8 +162,8 @@ <h4 class="text-muted">
                                     {# Organization #}
                                     <fieldset class="col-sm-6">
                                         <label>Organization *</label>
-                                        <input class="form-control form-required" {% if profile.organization %}value="{{ profile.organization }}"{% endif %} name="organization">
-                                        <small class="text-muted">The name of the organization you work for.</small>
+                                        <input class="form-control form-required tagsinput" {% if profile.organization %}value="{{ profile.organization }}"{% endif %} name="organization">
```

Comment:
We need to overwrite some CSS rules for this class in order to make the tags look more consistent with the styling of the site. This would me my take at it:

![image](https://user-images.githubusercontent.com/2448568/32675417-2360fe8c-c614-11e7-8f58-83db867fbe84.png)

Reply:
@Maurier That looks good. Can you send me your CSS?

Reply:
The following CSS rule should do the trick.
`.tagsinput, .tagsinput * {
    background: #DDD;
    color: #111;
}`


![image](https://user-images.githubusercontent.com/2448568/32970034-8e7b2a22-cba4-11e7-9dab-ca909cec6d41.png)